src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/EHInfo.cs

```diff
@@ -132,11 +132,11 @@ public void WriteTo(TextWriter writer, int methodRva, bool dumpRva)
             switch (Flags & CorExceptionFlag.COR_ILEXCEPTION_CLAUSE_KIND_MASK)
```

Comment:
There are bugs in the interpolated strings above as well. For example,, `writer.Write(@"(RVA {(HandlerEnd + methodRva):X4}) ")` should be `writer.Write($"(RVA {(HandlerEnd + methodRva):X4}) ")`.

Reply:
Also, `$@` can be just `$`

Reply:
Good catch, fixed.